samples/extensions/extended_dynamic_state2/extended_dynamic_state2.cpp

```diff
@@ -148,9 +152,18 @@ void ExtendedDynamicState2::render(float delta_time)
  */
 void ExtendedDynamicState2::prepare_uniform_buffers()
 {
-	uniform_buffers.common      = std::make_unique<vkb::core::Buffer>(get_device(), sizeof(ubo_common), VK_BUFFER_USAGE_UNIFORM_BUFFER_BIT, VMA_MEMORY_USAGE_CPU_TO_GPU);
-	uniform_buffers.baseline    = std::make_unique<vkb::core::Buffer>(get_device(), sizeof(ubo_baseline), VK_BUFFER_USAGE_UNIFORM_BUFFER_BIT, VMA_MEMORY_USAGE_CPU_TO_GPU);
-	uniform_buffers.tesselation = std::make_unique<vkb::core::Buffer>(get_device(), sizeof(ubo_tess), VK_BUFFER_USAGE_UNIFORM_BUFFER_BIT, VMA_MEMORY_USAGE_CPU_TO_GPU);
+	uniform_buffers.common      = std::make_unique<vkb::core::Buffer>(get_device(),
+																	  sizeof(ubo_common),
+																	  VK_BUFFER_USAGE_UNIFORM_BUFFER_BIT,
+																	  VMA_MEMORY_USAGE_CPU_TO_GPU);
+	uniform_buffers.baseline    = std::make_unique<vkb::core::Buffer>(get_device(),
+																	  sizeof(ubo_baseline),
+																	  VK_BUFFER_USAGE_UNIFORM_BUFFER_BIT,
+																	  VMA_MEMORY_USAGE_CPU_TO_GPU);
+	uniform_buffers.tesselation = std::make_unique<vkb::core::Buffer>(get_device(),
+																	  sizeof(ubo_tess),
+																	  VK_BUFFER_USAGE_UNIFORM_BUFFER_BIT,
+																	  VMA_MEMORY_USAGE_CPU_TO_GPU);
```

Comment:
Please run clang-format on the file, looks like sth is wrong with formatting of lines that were broken

Reply:
the changes are updated. please re-review the PR